src/social_auth/models.py

```diff
@@ -7,7 +7,9 @@
 from datetime import datetime, timedelta
 from django.conf import settings
 from django.db import models
-from django.db.models.loading import get_model
+from django.apps import apps
+
+get_model = apps.get_model
```

Comment:
Should we just use `apps.get_model` directly? Looks like it's just used in one place.